src/rendering/renderers/shared/background/BackgroundSystem.ts

```diff
@@ -111,6 +111,16 @@ export class BackgroundSystem implements System<BackgroundSystemOptions>
 
     set color(value: ColorSource)
     {
+        const incoming = new Color(value);
+
+        if (incoming.alpha < 1 && this._backgroundColor.alpha === 1)
+        {
+            console.warn(
+                '[PixiJS] ✨ Cannot set a transparent background on an opaque canvas. '
+            + 'To enable transparency, set backgroundAlpha < 1 when initializing your Application.'
+            );
+        }
+
```

Comment:
Can you use `import { warn } from '../../../../utils/logging/warn';` here which provides some convenience with warnings? We also wrap our warnings in these comment to exclude from minified releases.

```suggestion
        // #if _DEBUG
        const incoming = new Color(value);

        if (incoming.alpha < 1 && this._backgroundColor.alpha === 1)
        {
           warn(
                'Cannot set a transparent background on an opaque canvas. '
            + 'To enable transparency, set backgroundAlpha < 1 when initializing your Application.'
            );
        }
        // #endif

```